src_text/TCP_connections.txt

```diff
@@ -1,22 +1,22 @@
-TCP_connections is what takes care of handling multiple TCP client instances to establish a reliable connection via TCP relays to a friend. Only connecting to a friend with one relay would not be very reliable so this level of abstraction is needed to take care of this. For example it makes sure that if a relay goes down then the connection to the peer will not be impacted. This is done by connecting to the other peer with more than one relay.
+TCP_connections takes care of handling multiple TCP client instances to establish a reliable connection via TCP relays to a friend. Connecting to a friend with only one relay would not be very reliable, so TCP_connections provides the level of abstraction needed to manage multiple relays. For example, it ensures that if a relay goes down, the connection to the peer will not be impacted. This is done by connecting to the other peer with more than one relay.
 
 TCP_connections is above TCP_client and below net_crypto. 
 
-A TCP connection in TCP_connections is defined as a connection to a peer though one or more TCP relays. To connect to another peer with TCP_connections, a connection in TCP_connections to the peer with DHT public key X will be created. Some TCP relays which we know the peer is connected to will then be associated to that peer. Those relays will be the relays the peer has sent to us via the onion module if the peer isn't connected directly yet. The peer will also send some relays he is connected to directly once a connection is established however this is done by another module.
+A TCP connection in TCP_connections is defined as a connection to a peer though one or more TCP relays. To connect to another peer with TCP_connections, a connection in TCP_connections to the peer with DHT public key X will be created. Some TCP relays which we know the peer is connected to will then be associated to that peer. If the peer isn't connected directly yet, these relays will be the ones that the peer has sent to us via the onion module. The peer will also send some relays he is connected to directly once a connection is established however this is done by another module.
```

Comment:
>The peer will also send some relays he is connected to directly once a connection is established however this is done by another module.

* missing comma before `however`

@irungentoo 
* in which module is it described?